website/contributing/release-stable-minor.md

```diff
@@ -29,14 +27,26 @@ title: Release Stable Minor
 
 0.66 stable is out!
```

Comment:
Probably we should change the version in the whole article (currently we have the mix content mentioning 0.66 and 0.69) or just write the sentences in version-agnostic way (X.Y.Z).

Reply:
I think it might be better to use a specific version, let me change it

Reply:
maybe just use a `0.M.P` type of thing?

Reply:
tbh it's fine either way 👍

Reply:
If we want to use `0.M.P`, the better thing to do here would be to change it everywhere in the release steps. I don't mind doing it but maybe should be a different PR?